### PR TITLE
internal/runner: separate env lines by null

### LINKED ISSUE #154: internal/runner: `readEnvFromFile` cannot parse env variables with new lines
The function `readEnvFromFile` naively parses on new-lines, which of course will not support environment variables containing new lines. This is causing truncation for multiline environment variables, for example.

For now I guess we can use the `env -0` flag to separate by null terminators, instead of new lines.

In the future, though, we probably should properly (JSON)-encode environment variables. I imagine this can be wrapped up as part of #148 - probably we could expose a subcommand of `runme` to wrap around something like `godotenv`, which handles marshaling/parsing.

### PR DESCRIPTION
**Pull request by mxsdev**
Fixes #154

### CODE REVIEW

File: internal/runner/service_test.go [hunk 1]
```diff
@@ -568,19 +568,45 @@ func Test_runnerService(t *testing.T) {
 	t.Run("ExecuteLicenseCat", func(t *testing.T) {
 		t.Parallel()
 
+		session, _ := client.CreateSession(context.Background(), &runnerv1.CreateSessionRequest{})
```
Comment (adambabik) [inline internal/runner/service_test.go:571]: No reason not to check the error with `require.NoError`.

File: internal/runner/service_test.go [hunk 2]
```diff
@@ -568,19 +568,45 @@ func Test_runnerService(t *testing.T) {
...
 
+		err = stream.Send(&runnerv1.ExecuteRequest{
+			ProgramName: "bash",
+			Directory:   "../..",
+			Commands: []string{
+				"export LICENSE=$(cat LICENSE)",
+			},
+			SessionId: sessionID,
+		})
+		assert.NoError(t, err)
```
Comment (adambabik) [inline internal/runner/service_test.go:585]: I use `require` when there is no point in moving further if a condition is false. `assert` is used when it's ok to continue.

File: internal/runner/service_test.go [hunk 3]
```diff
@@ -568,19 +568,45 @@ func Test_runnerService(t *testing.T) {
 	t.Run("ExecuteLicenseCat", func(t *testing.T) {
```
Comment (adambabik) [inline internal/runner/service_test.go:568]:
```suggestion
	t.Run("ExecuteMultilineEnvExport", func(t *testing.T) {
```

File: internal/runner/service_test.go [hunk 4]
```diff
@@ -568,19 +568,45 @@ func Test_runnerService(t *testing.T) {
...
+				"export LICENSE=$(cat LICENSE)",
+			},
+			SessionId: sessionID,
+		})
+		assert.NoError(t, err)
+
+		{
+			execResult := make(chan executeResult)
+			go getExecuteResult(stream, execResult)
+			<-execResult
```
Comment (adambabik) [inline internal/runner/service_test.go:590]: It's worth to at least check the exit code with `require.EqualValues(t, 0, result.ExitCode)`.

**Comment by mxsdev** [comment]
@adambabik Good to merge?

**Review by adambabik — APPROVED** [review]

**Pull request merged**